lab_1/main.py

```diff
@@ -124,5 +216,37 @@ def sort_concordance(tokens: list, word: str, left_context_size: int, right_cont
     right_context_size = 3
     left_sort = True
     --> [['dog', 'is', 'happy', 'but', 'the', 'cat'], ['man', 'is', 'happy', 'the', 'dog', 'is']]
-    """
-    pass
+    В данной функции ОБЯЗАТЕЛЬНО использовать функцию get_concordance (см. Шаг 5).
+    """
+    word = word.lower()
+    operative_tokens = get_concordance(tokens, word, left_context_size, right_context_size)
+    concordance = []
+    sorted_concordance = []
+    #print(operative_tokens)
+
+    def left_sort (operative_tokens):
+        for i in operative_tokens:
+            for alfa in i[0]:
+                return alfa[0]
+
+    def right_sort(operative_tokens):
+        for i in operative_tokens:
+            index = len(i) - right_context_size
+            for alfa in i[index]:
+                return alfa[0]
+
+    if left_sort == True:
```

Comment:
The function `sort_concordance` has `left_sort` as an argument, and you redefine it as a function at the line №227, it will be overwritten.

Rename function and place it somewhere outside.